tensorflow/compiler/tf2tensorrt/kernels/trt_engine_op.cc

```diff
@@ -671,14 +672,18 @@ void TRTEngineOp::ComputeAsync(OpKernelContext* ctx,
     return;
   }
 
-  if (!use_implicit_batch_ && has_dynamic_shape_input_) {
+  if (!use_implicit_batch_ && has_dynamic_shape_input_ ||
```

Comment:
Clang doesn't like this, we need to add parentheses 
s/!use_implicit_batch_ && has_dynamic_shape_input_
/(!use_implicit_batch_ && has_dynamic_shape_input_
)/